### PR TITLE
LPS-118184

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi @natocesarrego ,

This is coming from natocesarrego#757, I've removed formatDate method since it is no longer used (and its test) and I've fixed testGetDDMFormFieldValues according to the logic.

Let me know if you have any question.

Thanks.

Regards.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java [hunk 1]
```diff
@@ -185,8 +188,6 @@ protected String getDDMFormFieldValue(
 			List<DDMFormInstanceRecord> ddmFormInstanceRecords, Locale locale)
 		throws Exception {
 
-		DateTimeFormatter dateTimeFormatter = getDateTimeFormatter(locale);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java:188]: Hey @antonio-ortega, can you please also remove this method and its references on tests? After your fix, it became useless as the method `formatDate`.

Thanks.

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/antonio-ortega/liferay-portal/pull/130. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/314f389a1e87~1...314f389a1e87) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**